pages/signup/index.js

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import { useForm } from "react-hook-form";
+import { useAccount } from "wagmi";
+import { object, string } from "yup";
+import { yupResolver } from "@hookform/resolvers/yup";
+import {
+  Button,
+  Container,
+  ErrorText,
+  Form,
+  Input,
+  Label,
+  Title,
+} from "./signUp.styles";
+
+const SignUp = () => {
+  const validationSchema = object().shape({
+    name: string().required(),
```

Comment:
I think it should be `username` instead of `name` to be able to have `isValid = true`. 
If not, I'm getting `isValid =  false`